app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+  alter_save :increment_posts_counter
```

Comment:
- [ ] [optional} Kindly check the spelling of the after_save you used l instead of f.